iml-task-runner/src/main.rs

```diff
@@ -28,6 +31,11 @@ const FID_LIMIT: i64 = 2000;
 // Number of seconds between cycles
 const DELAY: Duration = Duration::from_secs(5);
 
+// Default pool limit if not overridden by POOL_LIMIT
+lazy_static! {
+    static ref POOL_LIMIT: u32 = get_pool_limit().unwrap_or(8);
```

Comment:
Why 8 instead of 10?

Reply:
Targets tend to be multiples of 4, so 8 seemed to fit better.